sci-mathematics/cgal/cgal-4.10.2.recipe

```diff
@@ -0,0 +1,130 @@
+SUMMARY="A Computational Geometry Algorithms Library"
+DESCRIPTION="CGAL is a software project that provides easy access to efficient\
+ and reliable geometric algorithms in the form of a C++ library. CGAL is used \
+in various areas needing geometric computation, such as geographic information\
+ systems, computer aided design, molecular biology, medical imaging, computer\
+ graphics, and robotics.
+
+The library offers data structures and algorithms like triangulations, Voronoi\
+ diagrams, Boolean operations on polygons and polyhedra, point set processing,\
+ arrangements of curves, surface and volume mesh generation, geometry \
+processing, alpha shapes, convex hull algorithms, shape analysis, AABB and KD \
+trees..."
+HOMEPAGE="http://www.cgal.org/"
+COPYRIGHT="2005 Stanford University (USA)
+	2008,2011 GeometryFactory (France)
+	1999-2004 Max-Planck-Institute Saarbruecken (Germany)
+	1995-2004 Exact Computation Project
+	1997-2004, 2005-2008 ASCLEPIOS Project, INRIA Sophia-Antipolis (France)
+	1997-2004 Utrecht University (The Netherlands)
+	1997-2004 ETH Zurich (Switzerland)
+	1997-2004 Tel-Aviv University (Israel)"
+LICENSE="GNU GPL v3
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-$portVersion/CGAL-$portVersion.tar.xz"
+CHECKSUM_SHA256="5cc666e9813cb6515c1b4d14a167160be24d60c43c97fef8970dd3798a4c66a8"
+SOURCE_DIR="CGAL-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandSuffix=$secondaryArchSuffix
+	commandBinDir=$binDir
+else
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	cgal$secondaryArchSuffix = $portVersion
+	cmd:cgal_create_cmakelists
+	cmd:cgal_create_cmake_script
+	cmd:cgal_make_macosx_app
+	lib:libCGAL$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Core$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_ImageIO$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Qt5$secondaryArchSuffix = 13.0.0 compat >= 13
```

Comment:
Since all libs seem to have the exact same version, shall we define `libVersion` and `libVersionCompat`?
```diff
 fi
+
+libVersion="13.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	cgal$secondaryArchSuffix = $portVersion
 	cmd:cgal_create_cmakelists
 	cmd:cgal_create_cmake_script
 	cmd:cgal_make_macosx_app
-	lib:libCGAL$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL$secondaryArchSuffix = $libVersionCompat
-	lib:libCGAL_Core$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Core$secondaryArchSuffix = $libVersionCompat
-	lib:libCGAL_ImageIO$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_ImageIO$secondaryArchSuffix = $libVersionCompat
-	lib:libCGAL_Qt5$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Qt5$secondaryArchSuffix = $libVersionCompat
```

BTW, since there are no recipes using `lib:libCGAL`, it would be nice to drop the recipe for the previous release, `cgal-4.9.1.recipe`.
(I did a case-non-sensitive search to check this.)